[AUTOMATE] Update Ballerina Standard Library (http) Specifications (#5380)
ballerina-platform/ballerina-dev-website: spec/http/spec.md

```

> This `Links` is generated from the `linkedTo` field in the `ResourceConfig` annotation, and added either to the 

Reviewer: ```suggestion
This `Links` is generated from the `linkedTo` field in the `ResourceConfig` annotation and added either to the 
```

> payload or as a `Link` header depends on the payload type. This `Links` will not be added when a `http:Response` is 

Reviewer: ```suggestion
payload or as a `Link` header depending on the payload type. This `Links` will not be added when an `http:Response` is 
```

```

#### 7.2.2 Links in the response

> The static `Links` generated from the `linkedTo` field will be injected to the json payload when it is not 

Reviewer: ```suggestion
The static `Links` generated from the `linkedTo` field will be injected into the JSON payload when it is not 
```

a closed record and not `readonly`. Suppose the user returns the below record type, the runtime will inject the `Links` 

> record as in the latter. So the response should be considered as a record with `Links` field.

Reviewer: ```suggestion
record as in the latter. Therefore, the response should be considered as a record with the `Links` field.
```
